firmware/README.md

```diff
@@ -0,0 +1,61 @@
+# Introduction
+
+Open-TL866 operates using *modes*. Each mode supports one or
+more target chips. A mode is used by uploading the .hex file
+for that mode to the TL866 (see
+[parent README](../README.md)), and then running the
+corresponding Python command line tool from the `py/otl866`
+directory.
+
+# Supporting a new mode
+
+To support a new mode:
+
+1.  Create a directory in `firmware/modes` for your new mode.
```

Comment:
these are all showing up as "1." instead of say "1. and 2." Is this supposed to be an numbered list?

Reply:
Under git preview these are working. You might have a borked previewer :)